spec/controllers/reports/regions_controller_spec.rb

```diff
@@ -214,6 +214,10 @@
     end
 
     context "when region has diabetes management enabled" do
+      before :each do
```

Comment:
You can drop the `:each`

Reply:
I think we need to keep it because we have to disable the feature flag - `:diabetes_management-reports` before every `it` block.

Reply:
Yup, but the default behavior of `before` is to do a `before :each`. We generally don't explicitly specify the `:each`.